cob_bringup/robots/cob4-1.launch

```diff
@@ -5,17 +5,17 @@
 	<arg name="pkg_env_config" default="$(find cob_default_env_config)"/>
```

Comment:
Please comment out also the arguments,  if you don' t define a ROBOT_ENV variable you will get an error and you are not using it...